TransferGuide.md

```diff
@@ -70,17 +70,17 @@ Now that you have two accounts created, a transfer can be prepared. For this tra
 
 #### Request
 ```
```

Comment:
Hey shashi i dont think your documentation is correct, the example payloads I believe may be different also I'm not entirely sure the endpoints exist. If you were not supposed to remove it then its ok

Reply:
removed the file